scripts/build-toolchains.sh

```diff
@@ -110,7 +110,7 @@ else
         esac; ) || die 'obsolete make version; need GNU make 4.x or later'
 
     module_prepare riscv-gnu-toolchain qemu
-    module_build riscv-gnu-toolchain --prefix="${RISCV}"
+    module_build riscv-gnu-toolchain --prefix="${RISCV} --enable-multilib"
```

Comment:
`--enable-multilib` should be outside of the quoted string, hence the CI failure.

Reply:
Clearly I did not do any testing :)